gateway_test.go

```diff
@@ -27,13 +26,15 @@ func TestMain(m *testing.M) {
 	if err != nil {
 		panic(err)
 	}
+	WriteDefaultConf(&config)
+	config.AppPath, err = ioutil.TempDir("", "tyk-test-")
 	config.Storage.Port, _ = strconv.Atoi(s.Port())
+	config.EnableAnalytics = true
```

Comment:
Previously this option was enabled only for 2 tests, did you enabled it for all for purpose?